src/Hosting/Hosting/src/Internal/WebHost.cs

```diff
@@ -158,8 +162,6 @@ public virtual async Task StartAsync(CancellationToken cancellationToken = defau
             // Fire IApplicationLifetime.Started
             _applicationLifetime?.NotifyStarted();
 
-            // Fire IHostedService.Start
-            await _hostedServiceExecutor.StartAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
Can't change this, it's quite breaking.

Reply:
The only reason I tried this was that @davidfowl seemed open to changing WebHost in https://github.com/aspnet/AspNetCore/issues/3774#issuecomment-501794264

> Another option might be an interface like IHostedService? We can change the WebHost to make sure these run before the server starts (I believe it doesn't today).

Reply:
Can't? We can discuss how breaking it is.